inc/classes/subscriber/Media/class-webp-subscriber.php

```diff
@@ -154,7 +154,15 @@ public function convert_to_webp( $html ) {
 		}
 
 		if ( ! $http_accept || false === strpos( $http_accept, 'webp' ) ) {
-			return $html;
+			$user_agent = isset( $this->server['HTTP_USER_AGENT'] ) ? $this->server['HTTP_USER_AGENT'] : '';
+
+			if ( $user_agent && preg_match( '#Firefox/(?<version>[0-9]{2})#i', $this->server['HTTP_USER_AGENT'], $matches ) ) {
```

Comment:
With the current pace of version increment, I think it is safer to allow versions > 99.
```suggestion
			if ( $user_agent && preg_match( '#Firefox/(?<version>[0-9]{2,})#i', $this->server['HTTP_USER_AGENT'], $matches ) ) {
```